pages/podcast.js

```diff
@@ -76,7 +77,12 @@ class Podcast extends React.Component {
 
                   return (
                     <Card data-testid="Podcast Card" className={styles.podcastCard} key={name}>
-                      <h3>{interviewee}</h3>
+                      <Heading
+                        text={interviewee}
+                        theme="white"
+                        customAnchorClass="anchorMarginPodcast"
```

Comment:
What's the point of this?

Reply:
The podcast page requires a different top margin otherwise it'll look cropped at the top.